pkg/security/ebpf/kernel/kernel.go

```diff
@@ -326,8 +326,13 @@ func (k *Version) HaveMmapableMaps() bool {
 }
 
 // HaveRingBuffers returns whether the kernel supports ring buffer.
+// https://github.com/torvalds/linux/commit/457f44363a8894135c85b7a9afd2bd8196db24ab
 func (k *Version) HaveRingBuffers() bool {
-	return features.HaveMapType(ebpf.RingBuf) == nil
+	if features.HaveMapType(ebpf.RingBuf) == nil {
+		return true
+	}
+
+	return k.Code >= Kernel5_8
```

Comment:
I'm failing to see why we need this change, I think relying only on the feature detection is much better here

Reply:
Updated 👍🏻